### DISCUSSION TITLE
👋 Help us improve learning materials in WinAppSDK & WinUI 3!

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/WindowsAppSDK#2985] [Feedback wanted]
Take this survey to let us know what features you want to learn more about in WinAppSDK & WinUI 3 and the best ways to share that info to you: https://ncv.microsoft.com/3f35MtWpQi

💙 Thanks in advance!

**Comment by dimohy** [comment]
I took a survey to convey my thoughts.

However, a portal update seems urgent. It's strange that 1.1 hasn't been reflected yet.
Is the productboard's portal no longer managed?
